gson/src/main/java/com/google/gson/annotations/JsonAdapter.java

```diff
@@ -87,6 +89,7 @@
  * @author Jesse Wilson
  */
 // Note that the above example is taken from AdaptAnnotationTest.
+@Documented
```

Comment:
I don't think it should be on this one. The adapter in use for a class or field is an implementation detail.